babel.config.js

```diff
@@ -1,14 +1,3 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-  plugins: [
-    [
-      'module-resolver',
-      {
-        alias: {
-          '@react-native-community/slider': './js/Slider.js',
```

Comment:
We likely still need the alias. I think removing this works because the repo uses RN 0.60, which still uses Haste and `@react-native-community/slider` is taken as a namespace from the package.json. Can you upgrade to 0.61  (which removed haste references) and see if it works?

Reply:
hi! I'm using the same setup without the babel resolver in [rn-google-signin](https://github.com/react-native-community/react-native-google-signin/blob/master/yarn.lock#L5087) with 0.61 and it works just fine.

Reply:
Hm, so how Node knows how to resolve the `@react-native-community/google-signin` package, if it's not defined in package.json and resolved by Babel? 🤔 I think Metro may still treat package.json as haste directories, or I'm missing something